src/main/java/com/techsophy/tsf/runtime/form/dto/FilterDTO.java

```diff
@@ -0,0 +1,12 @@
+package com.techsophy.tsf.runtime.form.dto;
+
+import lombok.Data;
+import java.util.Map;
+
+@Data
+public class FilterDTO
```

Comment:
What is this class for?

Reply:
sorry this class is not required i am deleting this class now